### PR TITLE
minor algo changes

### PR DESCRIPTION
**Pull request by hannahli23**
Changes
----

- Added pricing filter and modified wheelchair time

Screenshots
----

- 

Resolved issues
-----

Closes #

### CODE REVIEW

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`server`](/server) directory were modified; please update the server version.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8908c18ef9179e53812c370a1b24f7f319d9d394
</p>

File: server/routes/map.js [hunk 1]
```diff
@@ -27,13 +27,30 @@ router.get("/query", async function (req, res, next) {
 
     if("wheelchair" === modetemp || "walking" === modetemp) {
```
Comment (evan10s) [inline server/routes/map.js]:
```suggestion
    if ("wheelchair" === modetemp || "walking" === modetemp) {
```

File: server/routes/map.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ router.get("/query", async function (req, res, next) {
     const long = req.query.long;
 
     //calculated later
-    const timeLimit = req.query.tt * 60; // seconds
+    let timeLimit = req.query.tt * 60; // seconds
```
Comment (evan10s) [inline server/routes/map.js]: I think `req.query.tt` is a string here, I suggest converting it to a string

File: server/routes/map.js [hunk 3]
```diff
@@ -27,13 +27,30 @@ router.get("/query", async function (req, res, next) {
 
     if("wheelchair" === modetemp || "walking" === modetemp) {
         mode = "walking";
+        if("wheelchair" === modetemp) {
```
Comment (evan10s) [inline server/routes/map.js]:
```suggestion
        if ("wheelchair" === modetemp) {
```

**Review by evan10s — APPROVED** [review]
Main thing is just to run req.query.tt through parseInt since I'm pretty sure that's a string (all URL parameters are string type).

but feel free to merge once the tests pass

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`server`](/server) directory were modified; please update the server version.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 524a3ba65fd827c36a9ac9d0f114c985ee957b36
</p>

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`server`](/server) directory were modified; please update the server version.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 406ae6c1089f3ab499902e2a40b21fc5c5071792
</p>

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`server`](/server) directory were modified; please update the server version.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e086442a7c1b95bae8269606f4b77e6f3a1fbee1
</p>

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`server`](/server) directory were modified; please update the server version.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e086442a7c1b95bae8269606f4b77e6f3a1fbee1
</p>

File: server/routes/map.js [hunk 4]
```diff
@@ -25,23 +25,40 @@ router.get("/query", async function (req, res, next) {
     const modetemp = req.query.transit;
     let mode = "";
 
-    if("wheelchair" === modetemp || "walking" === modetemp) {
+    if ("wheelchair" === modetemp || "walking" === modetemp) {
```
Comment (evan10s) [inline server/routes/map.js]:
```suggestion
    if ("wheelchair_or_scooter" === modetemp || "walking" === modetemp) {
```

File: server/routes/map.js [hunk 5]
```diff
@@ -25,23 +25,40 @@ router.get("/query", async function (req, res, next) {
...
         mode = "walking";
+        if ("wheelchair" === modetemp) {
```
Comment (evan10s) [inline server/routes/map.js]:
```suggestion
        if ("wheelchair_or_scooter" === modetemp) {
```

File: server/routes/map.js [hunk 6]
```diff
@@ -25,23 +25,40 @@ router.get("/query", async function (req, res, next) {
...
+            timeLimit = timeLimit/2;
```
Comment (evan10s) [inline server/routes/map.js:32]:
```suggestion
            timeLimit = timeLimit / 2;
```

**Pull request merged**